### PR TITLE
Changes for lazy loading module on client side

### PR DESCRIPTION
**Pull request by MrSpark2591**
This might be breaking changes as this will remove AWS object from global scope and we have to import it as we go.

### CODE REVIEW

File: client-tests.js [hunk 1]
```diff
@@ -1,16 +1,3 @@
...
-  try {
-    AWS;
-    isDefined = true;
-  }
-  catch (e) {
-  }
-  test.isTrue(isDefined, "AWS is not defined");
-  test.isTrue(Package['peerlibrary:aws-sdk'].AWS, "Package.peerlibrary:aws-sdk.AWS is not defined");
-
-  var s3 = new AWS.S3();
```
Comment (mitar) [inline client-tests.js]: Could you keep these tests?

File: client-tests.js [hunk 2]
```diff
@@ -1,16 +1,3 @@
...
-  test.isTrue(s3);
-  test.isTrue(s3.getObject);
-  test.isTrue(s3.upload);
-});
+Tinytest.add('aws-sdk', function(test) {
+  test.isFalse(Package['peerlibrary:aws-sdk'].AWS, "Package.peerlibrary:aws-sdk.AWS is not defined");
```
Comment (mitar) [inline client-tests.js]: Change `not defined` to `defined`.

File: .versions [hunk 3]
```diff
@@ -1,11 +1,11 @@
 allow-deny@1.1.0
```
Comment (mitar) [inline .versions]: Don't modify this file. It is modified when I publish the package.

File: server-tests.js [hunk 4]
```diff
@@ -1,18 +1,10 @@
...
-    isDefined = true;
-  }
-  catch (e) {
-  }
-  test.isTrue(isDefined, "AWS is not defined");
+Tinytest.add('aws-sdk', function(test) {
   test.isTrue(Package['peerlibrary:aws-sdk'].AWS, "Package.peerlibrary:aws-sdk.AWS is not defined");
 
-  var s3 = new AWS.S3();
+  var s3 = new Package['peerlibrary:aws-sdk'].AWS.S3();
```
Comment (mitar) [inline server-tests.js]: Why not importing the package?

File: client-tests.js (hunk 1 above)
Comment (MrSpark2591) [reply client-tests.js]: For lazy loading it shouldn't get imported. That's why i have removed these test. Also i was unable to find out how to test out lazy loading package. Please guide me.

File: client-tests.js (hunk 2 above)
Comment (MrSpark2591) [reply client-tests.js]: This will be resolved if we are able to find out a way to test lazy loaded package.

File: .versions (hunk 3 above)
Comment (MrSpark2591) [reply .versions]: yes sure i will revert my changes.

File: server-tests.js (hunk 4 above)
Comment (MrSpark2591) [reply server-tests.js]: I will import it and submit PR.

**Comment by MrSpark2591** [comment]
@mitar I have reverted version bumping and changed PR as per your input. Please publish package once this branch has been merged with version bumping.

**Comment by MrSpark2591** [comment]
@mitar Is it possible to publish the package so that I can change my dependency to this package instead of my forked one?

**Comment by mitar** [comment]
Published as `2.598.0_1`. In fact there was no easy way to demonstrate the backwards incompatibility of this change. :-(

**Pull request merged**